compiler/front/scriptconfig.nim

```diff
@@ -254,6 +254,20 @@ proc runNimScript*(cache: IdentCache; scriptName: AbsoluteFile;
   var m = graph.makeModule(scriptName)
   incl(m.flags, sfMainModule)
   var vm = setupVM(m, cache, scriptName.string, graph, idgen)
+  let isDisallowDanger =
```

Comment:
```suggestion
  let disallowDanger =
```

not strictly necessary, but it was done in `vm.nim` and it reads a tiny bit better.